.genx/configure.js

```diff
@@ -1,5 +1,6 @@
 const versions = require('./versions.json');
-const { registerPartials, generateRoute, generateEmptyCsv, formatRouteData, validateRoute } = require('./utils');
```

Comment:
To me `angular support` is when it'll match the normal client folder in terms of functionality (routes, FDC3, SSO etc.). This one I'd either call basic/initial Angular support, or put in draft until those other bits are done.

Reply:
renamed, makes sense